### ISSUE TITLE
Fix angular UI

### ISSUE THREAD

**Issue by webknjaz** [GDG-Ukraine/gdg.org.ua#71]
I've mentioned inability to set the date in admin UI. Need to fix this and check for any other inconsistencies.

**Comment by anxolerd** [comment]
@webknjaz: please provide the corresponding url and screenshots. If there is any error log, please attach it to the issue as well.

**Comment by webknjaz** [comment]
@anxolerd If you open event and modify date, you'll mention validation errors in console.

**Comment by webknjaz** [comment]
```
TypeError: s.test is not a function
    at http://localhost:8080/js/angular-strap.min.js:8:4847
    at https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:151:379
    at Array.forEach (native)
    at n (https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:6:470)
    at $setViewValue (https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:151:349)
    at http://localhost:8080/js/angular-strap.min.js:8:5142
    at Object.e.$eval (https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:92:272)
    at Object.e.$apply (https://ajax.googleapis.com/ajax/libs/angularjs/1.1.5/angular.min.js:92:379)
    at HTMLInputElement.<anonymous> (http://localhost:8080/js/angular-strap.min.js:8:5122)
    at HTMLInputElement.b.event.dispatch (http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js:3:28337)
```

**Comment by webknjaz** [comment]
Fixed by fd6a50b25a5c397affa3a7cb0f6ae5047c0e6995
